README.md

```diff
@@ -25,98 +50,75 @@ The following helpers are supported and automatically selected, if present, in t
 |skip_pgp_check |yes, **no**                                                  |Only valid with makepkg. Skip PGP signatures verification of source file, useful when installing packages without GnuPG properly configured.|
 |ignore_arch    |yes, **no**                                                  |Only valid with makepkg. Ignore a missing or incomplete arch field, useful when the PKGBUILD does not have the arch=('yourarch') field.|
 
-### Note
+#### Note
 * Either *name* or *upgrade* is required, both cannot be used together.
 * In the *use*=*auto* mode, makepkg is used as a fallback if no known helper is found.
 
-## Installing
-### AUR package
-The [ansible-aur-git](https://aur.archlinux.org/packages/ansible-aur-git) package is available in the AUR.
-
-Note: The module is installed in `/usr/share/ansible/plugins/modules` which is one of the default module library paths.
-
-### Manual installation
-Just clone the *ansible-aur* repository into your user custom-module directory:
-```
-git clone https://github.com/kewlfft/ansible-aur.git ~/.ansible/plugins/modules/aur
-```
-
-### Ansible Galaxy
-*ansible-aur* is available in Galaxy which is a hub for sharing Ansible content. To download it, use:
-```
-ansible-galaxy install kewlfft.aur
-```
-
-Note: If this module is installed from Ansible Galaxy, you will need to list it explicitly in your playbook:
-```
-# playbook.yml
-- hosts: localhost
-  roles:
-  - kewlfft.aur
-  tasks:
-  - aur: name=package_name
-```
-
-or in your role:
-```
-# meta/main.yml
-dependencies:
-- kewlfft.aur
-```
-
-```
-# tasks/main.yml
-- aur: name=package_name
-```
-
-## Usage
-### Notes
+### Usage
+#### Notes
 * The scope of this module is installation and update from the AUR; for package removal or for updates from the repositories, it is recommended to use the official *pacman* module.
 * The *--needed* parameter of the helper is systematically used, it means if a package is up-to-date, it is not built and reinstalled.
 
-### Create the "aur_builder" user
+#### Create the "aur_builder" user
 While Ansible expects to SSH as root, makepkg or AUR helpers do not allow executing operations as root, they fail with "you cannot perform this operation as root". It is therefore recommended to create a user, which is non-root but has no need for password with pacman in sudoers, let's call it *aur_builder*.
 
 This user can be created in an Ansible task with the following actions:
 ```
-- user:
+- name: Create the `aur_builder` user
+  ansible.builtin.user:
```

Comment:
is it only to avoid potential conflict? would `user:` still work?

Reply:
Yes, exactly. I changed the module names to FQCNs so we were in line with Ansible's documentation and best practices.

Reply:
Also, I will reply to your other comment later.

Thanks,
Maxwell 😀